### PR TITLE
Implementation of FlickrPhotosImporter

### PR DESCRIPTION
**Pull request by seehamrun**
#213

### CODE REVIEW

File: extensions/data-transfer/portability-data-transfer-flickr/src/test/java/org/dataportabilityproject/datatransfer/flickr/photos/FlickrTestUtils.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+package org.dataportabilityproject.datatransfer.flickr.photos;
+
+import com.flickr4java.flickr.photos.Photo;
+import com.flickr4java.flickr.photos.Size;
+import com.flickr4java.flickr.photosets.Photoset;
+
+import java.util.Collections;
+
+public class FlickrTestUtils {
```
Comment (rtannenbaum) [inline extensions/data-transfer/portability-data-transfer-flickr/src/test/java/org/dataportabilityproject/datatransfer/flickr/photos/FlickrTestUtils.java]: can this be package private?

File: portability-spi-transfer/src/main/java/org/dataportabilityproject/spi/transfer/types/TempPhotosData.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import org.dataportabilityproject.types.transfer.models.DataModel;
+import org.dataportabilityproject.types.transfer.models.photos.PhotoAlbum;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@JsonTypeName("org.dataportability:TempPhotosData")
+public class TempPhotosData extends DataModel {
```
Comment (rtannenbaum) [inline portability-spi-transfer/src/main/java/org/dataportabilityproject/spi/transfer/types/TempPhotosData.java]: Can you document this class?

File: extensions/data-transfer/portability-data-transfer-flickr/src/test/java/org/dataportabilityproject/datatransfer/flickr/photos/FlickrTestUtils.java (hunk 1 above)
Comment (seehamrun) [reply extensions/data-transfer/portability-data-transfer-flickr/src/test/java/org/dataportabilityproject/datatransfer/flickr/photos/FlickrTestUtils.java]: Done

File: portability-spi-transfer/src/main/java/org/dataportabilityproject/spi/transfer/types/TempPhotosData.java (hunk 2 above)
Comment (seehamrun) [reply portability-spi-transfer/src/main/java/org/dataportabilityproject/spi/transfer/types/TempPhotosData.java]: Done.

**Pull request merged**